lib/validLabels.js

```diff
@@ -1,9 +1,19 @@
-export default {
-    bug: 'Bug Fixes',
-    upgrade: 'Dependency Upgrades',
-    documentation: 'Documentation',
-    feature: 'Features',
-    enhancement: 'Enhancements',
-    build: 'Build-Related',
-    breaking: 'Breaking Changes'
-};
+import path from 'path';
+
+let validLabels;
+
+try {
+    validLabels = require(path.join(process.cwd(), 'pr-log.json'));
```

Comment:
Using `require` directly makes this hard to test. One (ugly) solution would be to use proxyquire. My preferred way would be to make this module export a function which uses dependency injection for the side-effecting read function.

For example:

```js
module.exports = function getValidLabels(readJsonFile) {
    let validLabels;
    try {
        validLabels = readFromJson(path.join(process.cwd(), 'pr-log.json'));
    } catch (err) { /* ... */ }
    return validLabels;
};
```

This would you allow to inject a stub when calling this in a test and in the actual code you can just inject `require`.